ci-operator/step-registry/ipi/conf/aws/metadata-service-auth/ipi-conf-aws-metadata-service-auth-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: ipi-conf-aws-metadata-service-auth
+  commands: ipi-conf-aws-metadata-service-auth-commands.sh
+  from_image:
+    namespace: ocp
+    name: "4.5"
+    tag: upi-installer
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: |-
+    The IPI AWS configure step generates the AWS-specific install-config.yaml contents based on the cluster profile and optional input files.
```

Comment:
Neither the cluster profile nor input files come into this step, do they?  It's just `AWS_METADATA_SERVICE_AUTH`?